SearchEngines.yml

```
@@ -751,10 +754,10 @@ Findhurtig:
 Fireball:
   -
     urls:
-      - www.fireball.de
+      - fireball.com
```

The first URL is used by Matomo for certain things. If we change it we would need to update a couple of things there. So would be easier to also keep `www.fireball.de`. As it redirect to `fireball.com` I don't think that's a problem...
```suggestion
      - www.fireball.de
      - fireball.com
```